src/apps/explorer/components/OrderWidget/index.tsx

```diff
@@ -21,10 +33,12 @@ export const OrderWidget: React.FC = () => {
   }, [orderId])
 
   return (
-    <div>
-      <h1>Details for order id: {orderId}</h1>
-      {order && <p>{JSON.stringify(order)}</p>}
+    <Wrapper>
+      <h2>Order details</h2>
+      {order ? <OrderDetails order={order} /> : <p>Order not found</p>}
+      <h2>Order fills</h2>
+      <p>No fills</p>
```

Comment:
placeholder?

Reply:
This part is not yet finalized in the mock ups, will implement it later.
Or do you mean something else?

Reply:
all good :+1: